sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -52,7 +52,7 @@ internal static LusidInstrument CreateExampleFxOption(bool isDeliveryNotCash = t
         internal static LusidInstrument CreateExampleEquityOption(bool isCashSettled = false)
             => new EquityOption(
                 startDate: new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero),
-                optionMaturityDate: new DateTimeOffset(2020, 12, 19, 0, 0, 0, TimeSpan.Zero),
+                optionMaturityDate: new DateTimeOffset(2020, 12, 19, 0, 0, 0, TimeSpan.Zero), // note that this is a weekend; lusid may skip over this date when generating valuation schedules
```

Comment:
for a user we need to define what the expected behaviour is, "may" is too ambiguous

Reply:
clarified a bit